pkg/policies/opa/rego/aws/aws_lambda_function/AWS.LambdaFunction.Logging.Low.0470.json

```diff
@@ -6,7 +6,7 @@
     },
     "severity": "LOW",
     "description": "Lambda tracing is not enabled.",
-    "reference_id": "AWS.LambdaFunction.Logging.Medium.0470",
+    "reference_id": "AWS.LambdaFunction.Logging.Low.0470",
```

Comment:
I think it's best to leave the severity out of the reference_id completely... The newer policies should be doing this; the older ones still need to be scrubbed.